.github/actions/yarn-install/action.yaml

```diff
@@ -39,4 +39,4 @@ runs:
         HUSKY: '0' # By default do not run HUSKY install
       shell: bash
       run: |
-        yarn install --immutable --inline-builds
+        yarn install --immutable --inline-builds
```

Comment:
I'd also revert this change whatever it is 😄

Reply:
one more lines, will remove it lol